docker-compose.yml

```diff
@@ -15,189 +18,287 @@ services:
+      ccdc-csd-deposition:
+        condition: service_healthy
+      ccdc-csd-crystal-structure-export:
+        condition: service_healthy
+    image: ccdcrepository.azurecr.io/onsite/csd-request-entry-api:4.2.0
+    environment:
+      - Logging__LogLevel__Default=Warning
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-formulasearch:
     labels:
       <<: *k8s-labels
     image: ccdcrepository.azurecr.io/onsite/ccdc-csd-formulasearch-api:4.0.0
     environment:
       - FormulaSearchReadConnection=${DB_CONNECTIONSTRING};SearchPath=formula;Database=${CSD_DATABASE}
+      - Logging__LogLevel__Default=Warning
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-unitcellsearch:
     labels:
       <<: *k8s-labels
     depends_on:
-      - ccdc-csd-reducedcell-calculation-service
-    image: ccdcrepository.azurecr.io/onsite/ccdc-csd-unit-cell-search:4.0.0
-    environment:
-      - UnitCellSearchReadConnection=${DB_CONNECTIONSTRING};SearchPath=unitcell;Database=${CSD_DATABASE}
+      ccdc-csd-reducedcell-calculation-service:
+        condition: service_healthy
     restart: unless-stopped
-
-  ccdc-csd-resultstore:
-    labels:
-      <<: *k8s-labels
-    image: ccdcrepository.azurecr.io/onsite/ccdc-csd-resultstore-api:4.0.0
+    image: ccdcrepository.azurecr.io/onsite/ccdc-csd-unit-cell-search:4.2.0
     environment:
-    - ResultStoreConnectionString=${DB_CONNECTIONSTRING};SearchPath=searchstore;Database=${CSD_DATABASE}
-    restart: unless-stopped
+      - UnitCellDatabaseConfiguration__DefaultDatabase=csd
+      - UnitCellDatabaseConfiguration__Databases__0__Name=csd
+      - UnitCellDatabaseConfiguration__Databases__0__ReadConnectionString=${DB_CONNECTIONSTRING};SearchPath=unitcell;Database=${CSD_DATABASE}
+      - UnitCellDatabaseConfiguration__Databases__0__WriteConnectionString=${DB_CONNECTIONSTRING};SearchPath=unitcell;Database=${CSD_DATABASE}
+      - Logging__LogLevel__Default=Warning
+    logging:
+      driver: local
 
   ccdc-csd-similaritysearch:
     labels:
       <<: *k8s-labels
     depends_on:
-      - ccdc-csd-fingerprint
+      ccdc-csd-fingerprint:
+        condition: service_healthy
     image: ccdcrepository.azurecr.io/onsite/ccdc-csd-structure-similarity-search:4.0.0
     environment:
       - StructureSimilaritySearchReadConnection=${DB_CONNECTIONSTRING};SearchPath=similarity;Database=${CSD_DATABASE}
+      - Logging__LogLevel__Default=Warning
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-fingerprint:
     labels:
       <<: *k8s-labels
-    image: ccdcrepository.azurecr.io/onsite/ccdc-csd-fingerprint-service:4.0.0
+    image: ccdcrepository.azurecr.io/onsite/ccdc-csd-fingerprint-service:4.2.0
     restart: unless-stopped
+    logging:
+      driver: local
 
   ccdc-csd-substructure-filter:
     labels:
       <<: *k8s-labels
     image: ccdcrepository.azurecr.io/onsite/ccdc-csd-substructure-filter-service:4.0.0
```

Comment:
Do the services from here downwards not need an  image update?

Reply:
No these haven't had any changes so no need to bump the version. We used to update them anyway but people made the choice not to do that anymore